src/main/java/frc/robot/commands/ReadyShooter.java

```diff
@@ -0,0 +1,49 @@
+package frc.robot.commands;
+
+import java.util.function.DoubleSupplier;
+
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.shooter.ShooterInterpolation;
+import frc.robot.subsystems.shooter.ShooterSubsystem;
+import frc.robot.subsystems.swerve.SwerveDrive;
+
+public class ReadyShooter extends Command {
```

Comment:
Either name this `ReadyShooterCommand`, or change your stated naming conventions in the README for this repo.